tic-tac-toe/src/components/board/board.js

```diff
@@ -0,0 +1,30 @@
+import React, {Component} from 'react';
+import Square from '../square/square';
+import './board.css';
+
+class Board extends Component {
+    renderSquare(i) {
+        return (
+            <Square
+                value={this.props.squares[i]}
+                onClick={() => {this.props.onClick(i)}}
+                key={i}
+            />
+        );
+    }   
+    render() {
+        const outerClassName = this.props.outerClassName;
+        const innerClassName = "board";
```

Comment:
don't create a variable for constant value inside render, it makes sense to keep it in global context at all

Reply:
function Board({squares, onClick}) {
  const squareComponents = squares.map((square, index) => {
    return (
      <Square
        value={squares[index]}
        onClick={() => {onClick(index)}}
        key={index}
      />
    )
  });